splunk-adx-alert-addon/ta_splunkadx_alert_declare.py

```diff
@@ -0,0 +1,23 @@
+# encode = utf-8
```

Comment:
Generic comment

a) Is this file generated, if so should we check in this file ?
b) Since there is a lot of custom build process wrto splunk add-on, we should have a small doc on how to build and test this

Reply:
1. Yes, this is a self generated file, will be removing it in the next commit.
2. sure, will add all those details in README